### PR TITLE
release 0.7.0

### PR DESCRIPTION
**Pull request by pdmthorsrud**
## What was added/changed/fixed?
Release 0.7.0! 🎉 

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [0.7.0 UNRELEASED]
+## 0.7.0
```
Comment (claesgill) [inline CHANGELOG.md:3]:
```suggestion
## [0.7.0]
```
Earlier, we haven't removed the square brackets, but maybe we should?

Comment (dangernil) [inline CHANGELOG.md:3]:
```suggestion
## [0.7.0]
```

Comment (dangernil) [reply CHANGELOG.md:3]: I think we should stay consistent and not remove brackets

Comment (pdmthorsrud) [reply CHANGELOG.md:3]: That is straight up an error from me!

**Review by claesgill — APPROVED** [review]
# 🎉

**Review by dangernil — APPROVED** [review]

**Review by zhenik — APPROVED** [review]

**Pull request merged**